crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -44,100 +51,53 @@ pub async fn create(
     provider: &JsonRpcClient<HttpTransport>,
     path_to_scarb_toml: Option<Utf8PathBuf>,
     chain_id: FieldElement,
-    maybe_salt: Option<FieldElement>,
+    salt: Option<FieldElement>,
     add_profile: bool,
     class_hash: Option<String>,
+    from_keystore: bool,
+    keystore_path: Option<Utf8PathBuf>,
+    account_path: Option<Utf8PathBuf>,
 ) -> Result<AccountCreateResponse> {
-    let private_key = SigningKey::from_random();
-    let public_key = private_key.verifying_key();
-    let salt = extract_or_generate_salt(maybe_salt);
-    let oz_class_hash: &str = if let Some(value) = &class_hash {
-        value
+    let (account_json, max_fee) = if from_keystore {
+        let keystore_path_ = keystore_path
+            .ok_or_else(|| anyhow!("--keystore must be passed when using --from-keystore"))?;
+        let account_path_ = account_path
+            .ok_or_else(|| anyhow!("--account must be passed when using --from-keystore"))?;
+        import_from_keystore(provider, keystore_path_, account_path_).await?
```

Comment:
I think `account create` should actually create an account; this looks to me that this is only importing, so should rather be done under https://github.com/foundry-rs/starknet-foundry/issues/434

Reply:
`account add --from-keystore` might be the place